template/hapi/server.js

```diff
@@ -18,21 +20,20 @@ server.ext('onPreResponse', function(request, reply) {
         var response = request.response;
         return reply(response.state('config', encodeURIComponent(internals.clientConfig)));
     }
-    else {
-        return reply();
-    }
+
+    return reply.continue();
 });
 
 
 // require moonboots_hapi plugin
-server.pack.register({plugin: require('moonboots_hapi'), options: moonbootsConfig}, function (err) {
+server.register({ register: require('moonboots_hapi').register, options: moonbootsConfig }, function (err) {
```

Comment:
"an object or array of objects where each one is either:" ... "an object with the following:" ... "`register` - the plugin registration function."

moonboots_hapi follows the convention in the second bullet point.  If you'll check the server.js example in moonboots_hapi you'll see it sends the whole module, not the register function.